spec/rubocop/cop/style/empty_lines_around_begin_body_spec.rb

```diff
@@ -0,0 +1,200 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Style::EmptyLinesAroundBeginBody do
+  let(:config) { RuboCop::Config.new }
+  subject(:cop) { described_class.new(config) }
+
+  shared_examples :offense do |name, code, correction|
+    it "registers an offense for #{name} with a blank" do
+      inspect_source(cop, code)
+      expect(cop.offenses.size).to eq(1)
+    end
+
+    it "autocorrects for #{name} with a blank" do
+      corrected = autocorrect_source(cop, code)
+      expect(corrected).to eq(correction)
+    end
+  end
+
+  shared_examples :accepts do |name, code|
+    it "accepts #{name}" do
+      inspect_source(cop, code)
+      expect(cop.offenses).to be_empty
+    end
+  end
+
+  include_examples :offense, 'begin body starting', <<-CODE, <<-CORRECTION
+begin
```

Comment:
Same remark as from your other PR - use `strip_margin` so we can pretty up the heredocs.